docs/en/operations/server-configuration-parameters/_README.md

```diff
@@ -0,0 +1,65 @@
+# Autogenerated server settings
+
```

Comment:
This file now contains all server settings which were in `settings.md` but which are not in `system.server_settings`.